assets/js/modules/adsense/datastore/adblocker.js

```diff
@@ -63,8 +68,21 @@ export const controls = {
 		// If this throws, then the fetch request failed completely and we'll assume it was blocked.
 		try {
 			await fetch(
-				'/favicon.ico?google-site-kit=/adsense/pagead2.googlesyndication.com/pagead/js/adsbygoogle.js',
-				{ credentials: 'omit' }
+				addQueryArgs(
+					'/favicon.ico',
+					{
+						// The name of the parameter here doesn't really matter
+						// since adblockers look at the URL as a whole.
+						'google-site-kit': '/adsense/pagead2.googlesyndication.com/pagead/js/adsbygoogle.js',
+						// Add a timestamp for cache-busting.
+						timestamp: Date.now(),
+					}
+				),
```

Comment:
The only thing that concerns me is that `addQueryArgs` escapes arguments which means that `/adsense/pagead2.googlesyndication.com/pagead/js/adsbygoogle.js` will be escaped to `%2Fadsense%2Fpagead2.googlesyndication.com%2Fpagead%2Fjs%2Fadsbygoogle.js`. However, I don't think it should bother adblocker and it should still detect it. :thinking:

Reply:
Good catch! I found that certain adblockers such as uBlock Origin did not work as before when the value was encoded. I've removed the use of `addQueryArgs` which was mostly added to make the construction of the URL a bit easier to read. It now uses a simpler method and explicitly mentions the importance of not being encoded.